usd/src/usd_parser/USDStage.cc

```diff
@@ -79,7 +91,9 @@ namespace usd
 
       if (this->dataPtr->upAxis != "Y" && this->dataPtr->upAxis != "Z")
       {
-        throw std::range_error("Up axis should be 'Y' or 'Z'");
+        errors.emplace_back(
+          Error(ErrorCode::ELEMENT_INVALID, "Up axis should be 'Y' or 'Z'"));
```

Comment:
I think `ELEMENT_INVALID` is used to represent sdf errors, it might not be suitable for usd errors. I think we should add usd specific errors similar to https://github.com/ignitionrobotics/sdformat/pull/836.